data/pages.yml

```diff
@@ -200,6 +200,8 @@
       url: "configuring-your-app"
     - title: "Configuring Ember CLI"
       url: "configuring-ember-cli"
+    - title: "Preparing for Ember Upgrades"
+      url: "preparing-for-upgrade"
```

Comment:
The specific issue being addressed is issue https://github.com/emberjs/guides/issues/889 which is specific to ember-cli-deprecation workflow.  It looks like parts 1 and 2 are the upgrade itself, while part 3 overlaps with this PR.  I think the title "preparing..." alludes more towards getting the codebase ready, specifically handling deprecations, though I'm open to suggestions.  It sounds like the other content you mentioned could be in a separate guide page focused on the act of upgrading if one doesn't already exist.